deploy/ansible/create-cluster.yml

```diff
@@ -9,9 +9,25 @@
 - hosts: localhost
   connection: local
   gather_facts: True
+
   vars_files:
     - "../profiles/{{ profile }}.yml"
-    - "{{ lookup('env', 'CREDENTIAL_SECRETS') }}"
+
+  vars:
+    bucket: "{{ bucket }}"
+    secrets: /tmp/vault/{{ profile }}
+
+  pre_tasks:
+    - name: Download from S3
+      aws_s3:
+        bucket: "{{ bucket }}"
+        object: "vault/{{ profile }}"
+        dest: "{{ secrets }}"
+        mode: get
+
+    - name: Include vars
+      include_vars:
+        file: "{{ secrets }}"
```

Comment:
there must be also a post_task which removes secrets file from filesystem after playbook execution, so we don't store sensitive data at filesystem permanently (even encrypted)

Reply:
keeping encrypted file on local filesystem could be useful for debug.